docker-compose.doris.yml

```diff
@@ -0,0 +1,75 @@
+services:
+  web:
+    volumes:
+      - ./docker/doris/dhis-doris.conf:/opt/dhis2/dhis.conf:ro
+    depends_on:
+      doris-be:
+        condition: service_healthy
+    networks:
+      doris_network:
+        ipv4_address: 192.168.100.9
+
+  db:
+    networks:
+      doris_network:
+        ipv4_address: 192.168.100.8
+
+  doris-fe:
+    image: apache/doris:fe-${DORIS_VERSION}
+    networks:
+      doris_network:
+        ipv4_address: 192.168.100.10
+    ports:
+      - "9030:9030"
+      - "8030:8030"
+    environment:
+      - FE_SERVERS=fe1:192.168.100.10:9010
+      - FE_ID=1
+    volumes:
+      - ./docker/doris/postgresql.jar:/opt/apache-doris/fe/jdbc_drivers/postgresql.jar:ro
+      - ./docker/doris/postgresql-42.7.5.jar:/opt/apache-doris/fe/jdbc_drivers/postgresql-42.7.5.jar:ro
```

Comment:
Do both files need to be mounted, given that one is symlink? 🤔

Reply:
Wouldn't the file be missing if we just mount the symlink? And vice versa?

Reply:
But maybe a better solution would be to mount the file without the version :thinking: